_posts/2016/2016-01-15-setup-a-coreos-cluster-just-for-fun-on-centos-and-win.md

```diff
@@ -0,0 +1,262 @@
+- edit file `user-data` replace "https://discovery.etcd.io/NEW_TOKEN" with the url you get above
+```
+etcd2: 
+discovery: https://discovery.etcd.io/NEW_TOKEN
+```
+
+- after replacing the link it should looks like this
+```
+etcd2: 
+discovery: https://discovery.etcd.io/0850c01e29cb6d7dcd5d4e9123311704
+```
+
+### 3. start clusters
+now change to your powershell 3 and execute following commands
+```
+$ cd coreos-vagrant
+$ vagrant up
+```
+If everything works fine you should see three virtualbox windows and your shell should look like this:
+
+{% image 2016_01_coreos_winsetup__01.jpg %}{% endimage %}
+
+### 4. stop clusters
+to stop the clusters execute following commands in your powershell 3
+```
+$ cd coreos-vagrant
+$ vagrant halt
+```
+
+### 5. increase cluster count
+to increase the count of clusters we have to change the file 'config.rb'
+lets try it to set up coreos with 10 clusters just for fun
+
+lets change this
+```
+# Size of the CoreOS cluster created by Vagrant
+$num_instances=3
+```
+
+to this
+```
+# Size of the CoreOS cluster created by Vagrant
+$num_instances=10
+```
+
+{% image 2016_01_coreos_winsetup__02.jpg %}{% endimage %}
+
+now change to your powershell 3 and execute following commands
+```
+$ cd coreos-vagrant
+$ vagrant up
+```
+
+if everything works fine you should see something like this
+
+{% image 2016_01_coreos_winsetup__03.jpg %}{% endimage %}
+
+### 6. remove clusters
+to completely remove the clusters execute following commands in your powershell 3
+```
+$ cd coreos-vagrant
+$ vagrant destroy
+```
+
+# CentOS
+
+## Prerequisits
+
+### 1. install git
+```
+$ yum install git
+```
+
+###  2. install openssl-devel and  openssl
+```
+$ yum install openssl-devel openssl
+```
+
+### 3. install nano
+```
+$ yum install nano
+```
+
+### 4. install [Vagrant](https://www.vagrantup.com)
+see also (https://www.vagrantup.com/downloads.html)
+```
+$ cd
+$ mkdir downloads
+$ cd downloads
+$ wget https://dl.bintray.com/mitchellh/vagrant/vagrant_1.7.4_x86_64.rpm
+$ rpm -Uvh vagrant_1.7.4_x86_64.rpm
+```
+
+### 5. install [VirtualBox](https://www.virtualbox.org/)
+see also (http://tecadmin.net/install-oracle-virtualbox-on-centos-redhat-and-fedora/)
+```
+$ vim /etc/yum.repos.d/virtualbox.repo
+```
+insert following text:
+```
+[virtualbox]
+name=Oracle Linux / RHEL / CentOS-$releasever / $basearch - VirtualBox
+baseurl=http://download.virtualbox.org/virtualbox/rpm/el/$releasever/$basearch
+enabled=1
+gpgcheck=1
+gpgkey=http://download.virtualbox.org/virtualbox/debian/oracle_vbox.asc
+```
+save and exit
+```
+:wq
+```
+execute
+```
+$ rpm -Uvh http://ftp.jaist.ac.jp/pub/Linux/Fedora/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
+
+$ yum install gcc make patch  dkms qt libgomp
+
+$ yum install kernel-headers kernel-devel fontforge binutils glibc-headers glibc-devel
+
+$ export KERN_DIR=/usr/src/kernels/2.6.32-504.3.3.el6.x86_64
+
+$ yum install VirtualBox-5.0
+
+$ service vboxdrv setup
+```
+
+## Lets get started
```

Comment:
![bildschirmfoto 2016-01-14 um 14 56 05](https://cloud.githubusercontent.com/assets/10460958/12326116/0aaa8ca2-bacf-11e5-8e01-291af927bdfe.png)
Layout is strange. Please adapt.